Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.m

```diff
@@ -113,19 +72,20 @@ + (instancetype)vok_addWithDictionary:(NSDictionary *)inputDict forManagedObject
 
 + (void)vok_addWithArrayInBackground:(NSArray *)inputArray completion:(VOKManagedObjectsReturnBlock)completion
 {
-    [VOKCoreDataManager importArrayInBackground:inputArray
-                                       forClass:[self class]
-                                     completion:^(NSArray *arrayOfManagedObjectIDs) {
-                                         if (completion) {
-                                             // if there is no completion block there's no need to collect the new/updated objects
-                                             NSMutableArray *returnArray = [NSMutableArray arrayWithCapacity:arrayOfManagedObjectIDs.count];
-                                             NSManagedObjectContext *moc = [[VOKCoreDataManager sharedInstance] managedObjectContext];
-                                             for (NSManagedObjectID *objectID in arrayOfManagedObjectIDs) {
-                                                 [returnArray addObject:[moc objectWithID:objectID]];
-                                             }
-                                             completion([returnArray copy]);
-                                         }
-                                     }];
+    VOKCoreDataManager *coreDataManager = VOKCoreDataManager.sharedInstance;
+    [coreDataManager importArrayInBackground:inputArray
```

Comment:
Right.  Basically, the convenience methods in `NSManagedObject+VOKManagedObjectAdditions` are (now that `vok_entityName` has been rewritten) just wrappers around methods on the manager, so if one needs a manager-specific call, passing the manager in as an argument to a method on the managed object seemed uglier than calling the method on the manager with the managed object as an argument.

There probably should be an explicit note to the effect of "things in `NSManagedObject+VOKManagedObjectAdditions` use `VOKCoreDataManager.sharedInstance`."